src/editors/containers/VideoUploadEditor/hooks.js

```diff
@@ -14,7 +14,7 @@ export const {
 export const postUploadRedirect = (storeState, uploadType = 'selectedVideoUrl') => {
   const learningContextId = selectors.app.learningContextId(storeState);
   const blockId = selectors.app.blockId(storeState);
-  return (videoUrl) => navigateTo(`/course/${learningContextId}/editor/video/${blockId}?${uploadType}=${videoUrl}`);
+  return (videoUrl) => navigateTo(`/authoring/course/${learningContextId}/editor/video/${blockId}?${uploadType}=${videoUrl}`);
```

Comment:
Same thing here - don't hard-code `/authoring/` and you shouldn't actually need it at all.